### PR TITLE
Localytics custom dimensions and sessions regressions (WEBAPP-3610, WEBAPP-3707)

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/config.coffee [hunk 1]
```diff
@@ -88,6 +88,8 @@ z.config =
     'image/gif'
   ]
 
+  LOCALYTICS_SESSION_TIMEOUT: 180000 # 3 minutes in milliseconds
```
Comment (lipis) [inline app/script/config.coffee]:
```
3 * 60 * 1000
```

File: app/script/tracking/CustomDimension.coffee [hunk 2]
```diff
@@ -19,5 +19,6 @@
 window.z ?= {}
 z.tracking ?= {}
 
-z.tracking.DimensionName =
-  CONTACTS: 'contacts'
+z.tracking.CustomDimension =
```
Comment (bennycode) [inline app/script/tracking/CustomDimension.coffee]: Please mention https://dashboard.localytics.com/settings/apps?org_id=1145 here.

File: app/script/ui/WindowHandler.coffee [hunk 3]
```diff
@@ -25,9 +25,13 @@ class z.ui.WindowHandler
     @logger = new z.util.Logger 'z.ui.WindowHandler', z.config.LOGGER.OPTIONS
 
     @height = 0
-    @is_visible = true
     @width = 0
 
+    @is_visible = true
+    @lost_focus_interval_time = (z.config.LOCALYTICS_SESSION_TIMEOUT / 3)
```
Comment (lipis) [inline app/script/ui/WindowHandler.coffee]: no need for brackets

File: app/script/tracking/EventTrackingRepository.coffee [hunk 4]
```diff
@@ -21,7 +21,7 @@ z.tracking ?= {}
 
 LOCALYTICS =
   APP_KEY: '905792736c9f17c3464fd4e-60d90c82-d14a-11e4-af66-009c5fda0a25'
-  TRACKING_INTERVAL: 60000 # milliseconds
+  SESSION_INTERVAL: 60000 # milliseconds
```
Comment (lipis) [inline app/script/tracking/EventTrackingRepository.coffee:24]: Maybe here as well.. `60 * 1000`

**Pull request merged**